tests/3DMain.spec.ts

```diff
@@ -14,13 +14,8 @@ test.beforeEach(async ({ page }) => {
 });
 
 test.describe('3D main Test', async () => {
-  test('should render 3D main correctly.', async ({ page }) => {
-    await page.getByTestId('Layout').click();
-    await page
-      .locator('div')
-      .filter({ hasText: /^3D main$/ })
-      .first()
-      .click();
+  test('should render 3D main correctly.', async ({ page, mainToolbarPageObject }) => {
+    await mainToolbarPageObject.layoutSection.threeDMain.click();
```

Comment:
are these typed? meaning do we have autocomplete when we do mainToolbarPageObject.